packages/apollo-angular/src/utils.ts

```diff
@@ -65,16 +62,12 @@ export class ZoneScheduler implements SchedulerLike {
   }
 }
 
-// XXX: Apollo's QueryObservable is not compatible with RxJS
-// TODO: remove it in one of future releases
-// https://github.com/ReactiveX/rxjs/blob/9fb0ce9e09c865920cf37915cc675e3b3a75050b/src/internal/util/subscribeTo.ts#L32
-export function fixObservable<T>(obs: ObservableQuery<T>): Observable<ApolloQueryResult<T>>;
-export function fixObservable<T>(obs: AObservable<T>): Observable<T>;
-export function fixObservable<T>(
-  obs: AObservable<T> | ObservableQuery<T>,
-): Observable<ApolloQueryResult<T>> | Observable<T> {
-  (obs as any)[observable] = () => obs;
-  return obs as any;
+export function fromObservableQuery<TData>(
+  obsQuery: ObservableQuery<TData, any>,
+): Observable<ApolloQueryResult<TData>> {
+  return new Observable(subscriber => {
+    return obsQuery.subscribe(subscriber);
+  });
```

Comment:
FYI `ObservableQuery` implements the `InteropObservable` interface, so you should be able to simplify this to:

```ts
import { from } from "rxjs";

// ...

return from(obsQuery);
```